rust/fury-derive/Cargo.toml

```diff
@@ -25,7 +25,12 @@ rust-version.workspace = true
 proc-macro = true
 
 [dependencies]
-proc-macro2 = { default-features = false, version = "1.0.66" }
```

Comment:
Cargo will try to resolve the "latest" package version that meets the semver rule. Hence the last patch version in Cargo.toml doesn't take efforts, unless use a strict match like `version = "=1.0.66"`